sktime/forecasting/croston.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pandas as pd
+from sktime.forecasting.base._sktime import _SktimeForecaster
+from sktime.forecasting.base._sktime import _OptionalForecastingHorizonMixin
+
+
+DEFAULT_SMOOTHING = 0.1
+DEFAULT_ALPHA = 0.05
```

Comment:
isn't that defined already somewhere else, in `forecasting/base` perhaps?

Reply:
yes found it in forecasting.base._base